iferr/errors.go

```diff
@@ -87,3 +87,27 @@ func (ie *IfErr) WriteHTTP(err error, w http.ResponseWriter, codeArg ...int) boo
 
 	return false
 }
+
+type Opts struct {
```

Comment:
Please name the struct RespondOpts because maybe there will be other methods with options in the future.